crates/jxl-grid/src/simple_grid.rs

```diff
@@ -102,7 +102,7 @@ pub struct CutGrid<'g, Lane: Copy = f32> {
     width: usize,
     height: usize,
     stride: usize,
-    _marker: std::marker::PhantomData<&'g mut [Lane]>
+    _marker: std::marker::PhantomData<&'g mut [Lane]>,
```

Comment:
Oops I missed the trailing comma 😅 maybe rustfmt added this?

Reply:
Yes. For example, `blend.rs` has a lot of fixes from rustfmt, I try not to include them in the pull request, but `simple_grid.rs` does not have many.